docs-2.0/2.quick-start/0.FAQ.md

```diff
@@ -92,6 +92,14 @@ Got 1 rows (time spent 1235/1934 us)
 
 * The second number 1934 shows the time spent from the client's perspective, that is, the time it takes for the client from sending a request, receiving a response, and displaying the result on the screen.
 
+### Can I set `replica_factor` as an even number in `CREATE SPACE` (e.g., replica_factor = 2) ?
+
+NO.
+
+The storage service use the Raft protocol (quorum-based voting) to elect a leader replica. If `replica_factor` is two, that requires both two replicas alive to vote. The majority of 2 is 2 itself, not 1. And when one replica dies, the remaining one replica is not enough for an election. Your system can't work if one replica is down.
+
+The suggested number is 3 in a production environment and 1 in a test environment. Do not use an even number.
```

Comment:
for your reference:
The storage service uses the Raft protocol (quorum-based voting) to elect a leader replica. When replica_factor is 2, both replicas take part in voting. The majority of 2 is 2, not 1. And when one replica is out of service, the other one can not start an election. Thus your system can not work. 

We suggest that you set replica_factor to 3 for the production environment and 1 for the test environment. Do not use an even number.